### PR TITLE
bpo-27505: Add version_added to docs for module attributes

### PR DESCRIPTION
**Pull request by csabella**
https://bugs.python.org/issue27505

### CODE REVIEW

File: Doc/reference/datamodel.rst
Comment (berkerpeksag) [inline Doc/reference/datamodel.rst]: Can you add an empty line after this line?

File: Doc/reference/datamodel.rst
Comment (berkerpeksag) [inline Doc/reference/datamodel.rst]: Can you also add a link to the PEP?

**Review by berkerpeksag — CHANGES_REQUESTED** [review]

**Comment by csabella** [comment]
I have made the requested changes; please review again.

**Review by berkerpeksag — APPROVED** [review]

File: Doc/reference/datamodel.rst
Comment (berkerpeksag) [inline Doc/reference/datamodel.rst]: Sorry, I missed this earlier. `module.__class__` attribute exists in Python versions older than 3.5. From Python 3.4:

```py
>>> from types import ModuleType
>>> sys.modules['argparse'].__class__
<class 'module'>
>>> sys.modules['argparse'].__class__ = ModuleType
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __class__ assignment: only for heap types
```

So I think this should be changed to use `versionchanged` instead of `versionadded` and it should say that the `__class__` attribute is now writable.

**Review by berkerpeksag — CHANGES_REQUESTED** [review]

File: Doc/reference/datamodel.rst
Comment (csabella) [reply Doc/reference/datamodel.rst]: @berkerpeksag  Thanks for catching that!  I've also updated in #5321, which is the backport of only the __class__ doc change to 3.6.

**Comment by csabella** [comment]
I have made the requested changes; please review again

**Review by berkerpeksag — APPROVED** [review]
LGTM, thank you! I will wait for a bit to give @ncoghlan time to review before merging this.

**Review by ncoghlan — APPROVED** [review]

**Pull request merged**
